packages/core/src/utils.js

```diff
@@ -60,6 +60,35 @@ export function percyAutomateRequestHandler(req, percy) {
   req.body.buildInfo = percy.build;
 }
 
+// Returns the body for sendEvent structure
+export function percyBuildEventHandler(req, cliVersion) {
```

Comment:
You seem to be updating incoming request body object here, its usually not a good practice, its better to pluck the data that you want from incoming body and create a new object [ that automatically excludes unintended/malicious data